test/endtoend/operator/101_initial_cluster.yaml

```diff
@@ -8,13 +8,13 @@ metadata:
   name: example
 spec:
   images:
-    vtctld: vitess/lite:v15.0.2
-    vtgate: vitess/lite:v15.0.2
-    vttablet: vitess/lite:v15.0.2
-    vtorc: vitess/lite:v15.0.2
-    vtbackup: vitess/lite:v15.0.2
+    vtctld: vitess/lite:v15.0.2-mysql80
+    vtgate: vitess/lite:v15.0.2-mysql80
+    vttablet: vitess/lite:v15.0.2-mysql80
+    vtorc: vitess/lite:v15.0.2-mysql80
+    vtbackup: vitess/lite:v15.0.2-mysql80
```

Comment:
This should be ported over to `main` as well right?

Reply:
No, on main we are now using v16 images and testing upgrade from them. So they are already from 8.0 to 8.0. Actually on main even the minor version is not being upgraded.